onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -372,23 +404,36 @@ def run_onnxruntime(use_gpu, model_names, fp16, batch_sizes, sequence_lengths, r
             if ort_session is None:
                 continue
 
+            ort_output_names = []
+            last_state_buffer = None
```

Comment:
I suggest to move these buffer allocation to inference_ort_with_io_binding:
    def inference_ort_with_io_binding(ort_session, ort_inputs, result_template, repeat_times, ort_output_names,
                                  max_batch_size, max_seq_length, device):
    if buffer is None:
       buffer = allocate buffer based on max_batch_size and max_seq_length
   use buffer in io binding.